pkg/convert/pot_test.go

```diff
@@ -0,0 +1,297 @@
+package convert
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
```

Comment:
I suggest grouping these e.g.
```go
import (
  stdlib
  ...

  3rd party packages
  ...

  our packages
  ...
)
```
That way it is easier to follow imports
[jetbrains](https://stackoverflow.com/a/70668482/19159194)
[vscode](https://marketplace.visualstudio.com/items?itemName=aleksandra.go-group-imports)

Reply:
It's ok to group in two groups as we do now - stdlib and others. Easier to maintain. I've seen the same grouping (3 groups) practice in other projects, and it is out of control - hard to maintain and developers don't care, it's chaos. I don't see real benefits.

Reply:
Could you elaborate on the meaning of out of control ?

Reply:
Because there is vscode extension and jetbrains config that automatically does that. IMO in that way it looks cleaner somehow

Reply:
In general, there are no benefits, developers don't spend much time looking at the list of imported packages. It is easy to find imported packages anyway. It has not been enforced in other projects, and developers add imported packages to arbitrary groups.

I've used both approaches in different projects. The simplest one works the best, it is more consistent and easier for developers to maintain.

Before I gained my experience building projects in go, I would agree with your statement.